src/led_effect.py

```diff
@@ -822,6 +822,39 @@ def __init__(self,  **kwargs):
 
             self.frameCount = len(self.thisFrame)
 
+    #Cylon, single LED bounces from start to end of strip
+    class layerCylon(_layerBase):
+        def __init__(self,  **kwargs):
+            super(ledEffect.layerCylon, self).__init__(**kwargs)
+
+            self.paletteColors = colorArray(COLORS, self.paletteColors)
+
+            blank = [0.0, 0.0, 0.0, 0.0]
```

Comment:
Please use `[0.0] * COLORS` for that. Also I don't think this needs to be a variable here, as it is only used once.

Reply:
Done.